instance_config.yaml

```diff
@@ -105,7 +105,17 @@ val_username:
 val_transcript_create_url:
 val_video_transcript_status_url:
 
-
+# ---
+# Celery Info
+# ---
+celery_app_name:
+# can do multiple queues like so: foo,bar,baz
+celery_worker_queue: encode_worker,large_encode_worker
+celery_deliver_queue: deliver_worker
+celery_heal_queue: heal_queue
+celery_threads: 1
+
+onsite_worker: False
```

Comment:
@yro We have separated configuration into two yaml files. Please see https://github.com/edx/edx-video-pipeline/pull/66. You need to move/update these values in [static_config.yaml](https://github.com/edx/edx-video-pipeline/blob/master/static_config.yaml). Let us know if there are any issues.

Reply:
@muhammad-ammar OK! Some of these aren't static, but that's a smart move. I'll make the change.